### ISSUE TITLE
SpecialGraphics missing exceptioin for Overland levels in Area View

### ISSUE THREAD

**Issue by manikus** [grannypron/uaf#64]
The SpecialGraphics Special Ability is missing an exception for Overland levels. When Area View is chosen, the image (i.e. the compass) appears on top of the map.

**Comment by grannypron** [comment]
I'm trying to recreate this and remember how to do special graphics in general.  Do you happen to have a game that has this feature that I can use to debug?

**Comment by manikus** [comment]
Here's my current testing design. I just realized that I need to update the template and tutorial designs to use the compass (SpecialGraphics).
This is the link to the documentation on it:
http://manikus.reonis.com/Help/SASpecialGraphic.html

The attached design had to have some graphics and the executable removed to become small enough to attach. The graphics removed aren't needed, but if you want to add them, they are all in the template design.

[230204_testing_DESIGN.zip](https://github.com/grannypron/uaf/files/10609242/230204_testing_DESIGN.zip)

**Comment by grannypron** [comment]
![image](https://user-images.githubusercontent.com/19980049/216790412-0662d583-7422-4dd6-8e03-f8573ef6363f.png)

**Comment by grannypron** [comment]
I see this is happening as part of the SpecialGraphics event.  Do you see this as being a scripting fix or a programming fix?

**Comment by manikus** [comment]
This is both I think. There needs to be another exception for overlands (or bigpics or both), and then the designer needs to turn it on with the script. If this were only for a compass, it would be just in the code, but I do use this for other things in my designs, like for border overlays which I do  still want to show up in overlands. If this is not a clear answer, let me know.

**Comment by grannypron** [comment]
I'm not sure what you mean by "exception".  I could set the "NoCompass" flag in the case of the overland event, but I'm not sure about the other cases you describe.

**Comment by manikus** [comment]
The compass is an instance of SpecialGraphics. While we don't want to show the compass in the overland, I do want to have another instance of SpecialGraphics visible in the overland.
By "exception" I meant the list of letters in the docs - the designer would have another letter they could use and that letter would not allow that particular SpecialGraphic to apear in overlands.

**Comment by manikus** [comment]
I don't think this is an enhancement - I think its a bug. SpecialGraphics doesn't have a way to not show up in overlands. It needs a new exception (SpecialGraphics uses exceptions to not show up on screens.)
http://manikus.reonis.com/Help/SASpecialGraphic.html
